.github/workflows/main.yml

```diff
@@ -67,8 +67,8 @@ jobs:
           fi
           echo "docker_build=1" >> $GITHUB_OUTPUT;
         else
-          echo "Dockerfile was not modified. Using develop branch image."

```

Comment:
@dstndstn This change might be holding you up.

Reply:
thanks